### PR TITLE
Update version.cfg

### PR DESCRIPTION
**Pull request by dc-redeclipse**
Fix map name from parker to park
Add capture (ctf) to cargo16
Add new mostly AI generated sci-fi theme fitting names for bots

### CODE REVIEW

File: config/version.cfg [hunk 1]
```diff
@@ -26,60 +26,178 @@ version_maplist = [
...
+    Foster
+    Fox
+    Ghost
+    Gideon
+    Harper
+    Havoc
+    Hayden
+    Hunter
+    Ian
+    Icon
```
Comment (IceflowRE) [inline config/version.cfg:57]: Would exclude this one.

File: config/version.cfg [hunk 2]
```diff
@@ -26,60 +26,178 @@ version_maplist = [
...
+    Naya
+    Nia
+    Ophelia
+    Petra
+    Phoenix
+    Polaris
+    Qiana
+    Rebekah
+    Rhea
+    Riot
```
Comment (IceflowRE) [inline config/version.cfg:168]: Should be fine as itself, but probably remove to due to name clash with Riot Games.

**Comment by qreeves** [comment]
Okay, so my problems with this are:
- There are far fewer names than there used to be, this will result in much less variety.
- The lines should be combined as much as possible, like how it was, otherwise the list takes up way too much space and makes the file difficult to navigate.

File: config/version.cfg (hunk 2 above)
Comment (IceflowRE) [reply config/version.cfg:168]: Thats true, just wanted to point that out.

Comment (dc-de) [reply config/version.cfg:168]: Sure... good note! The AI is not perfect for sure.

**Comment by dc-de** [comment]
It should be now solved.
The AI is stupid, won't give me more useful names, but I think we have enough now.

**Comment by dc-de** [comment]
Fixed @qreeves
sorry, I forget to resolve this

**Review by dc-redeclipse — COMMENTED** [review]
more sci-fi names

**Comment by dc-redeclipse** [comment]
@qreeves the names should be now fine.
I add a bunch more fitting the sci-fi theme and cyber look.
Is it now ok?

**Comment by qreeves** [comment]
Yeah, but you'll need to rebase on current master as there's a conflict that's preventing merging.

**Pull request closed without merging**